bot/exts/fun/duck_pond.py

```diff
@@ -165,6 +176,10 @@ async def on_raw_reaction_add(self, payload: RawReactionActionEvent) -> None:
         message = await channel.fetch_message(payload.message_id)
         member = discord.utils.get(message.guild.members, id=payload.user_id)
 
+        # Was the message sent in a channel Helpers can see?
+        if not self.is_helper_viewable(channel):
+            return
```

Comment:
Move this before the message and member are retrieved. If this condition turns out to be false, those aforementioned operations would end up being wasteful.